contracts/connection-router/src/msg.rs

```diff
@@ -18,6 +18,7 @@ pub struct Message {
 #[cw_serde]
 pub struct InstantiateMsg {
     pub admin_address: String,
+    pub governance_address: String,
```

Comment:
We can probably add comments here explaining the role of each. Also, update the comment inside the `ExecuteMsg` enum, that says all of the below messages can only be called by the router admin.